lib/msal-common/src/app/config/ModuleConfiguration.ts

```diff
@@ -8,6 +8,11 @@ import { ICrypto, PkceCodes } from "../../crypto/ICrypto";
 import { AuthError } from "../../error/AuthError";
 import { ILoggerCallback, LogLevel } from "../../logger/Logger";
 
+/**
+ * Defaults for the Module Configuration Options
+ */
+const OFFSET = 300;
```

Comment:
Offset for what? Variable name should be more specific.

Reply:
Fixed in a future PR.